src/routes/chains/chains.controller.spec.ts

```diff
@@ -200,7 +200,7 @@ describe('Chains Controller (Unit)', () => {
       });
     });
 
-    it('Failure: received data is not valid', async () => {
```

Comment:
We should keep this as there may be issue with non-`results` as well.

Reply:
I added this back in a [separate PR](https://github.com/safe-global/safe-client-gateway/pull/1881).